solidity/random-beacon/test/helpers/contract-test-helpers.ts

```diff
@@ -0,0 +1,11 @@
+import { ethers } from "hardhat"
```

Comment:
What if we try to extract helpers to some common library, e.g. https://github.com/keep-network/hardhat-helpers?

Reply:
@nkuba I'm OK with keeping functions like `increaseTime` in the common library like `hardhat-helpers`.
@pdyraga What do you think?

Reply:
Yes, we are repeating this code in every repository. It is time to extract it.

Reply:
If you don't mind, we could do it in a separate PR.

Reply:
I did it in https://github.com/keep-network/hardhat-helpers/pull/5

Reply:
I used `hardhat-helpers` in https://github.com/keep-network/keep-core/pull/2613/commits/3506fdfc60ca6aed60af446299610f6fcd69a6f7.
I added the package vie the GitHub url with the latest commit hash:
"@keep-network/hardhat-helpers": "https://github.com/keep-network/hardhat-helpers#ed35ab7",

If we release the new version of `hardhat-helper` then we can replan it with that package instead of using GitHub url.